resources/ctl.rb

```diff
@@ -60,34 +60,6 @@
       version new_resource.version unless new_resource.version == 'latest'
     end
   end
-
-  if platform?('windows')
```

Comment:
are these temp directories still used? Should we offer an option to cleanup old files on existing/running agents?

Reply:
Directories wouldn't be used anymore and are cleaned up during the same chef run.

Reply:
Would it make sense to remove these and then remove the code in a future release?

Reply:
Do you mean the temp directories? They're cleaned up in the resource in line 85 in the old code, like Derek mentioned.